test/gc/test-net-timeout.js

```diff
@@ -35,7 +35,11 @@ function getall() {
   if (count >= todo)
     return;
 
-  const req = net.connect(server.address().port, server.address().address);
+  if (common.hasIPv6)
+   var host = '::1';
```

Comment:
Indentation is not right. Also, if we are doing this ternary operator is okay I guess